src/components/NewCharacter/NewCharacter.js

```diff
@@ -1,9 +1,10 @@
 import React, { useState } from 'react';
 import { string, func } from 'prop-types';
-import { characters } from 'gm-common';
+import { characters, characterProperties } from 'gm-common';
```

Comment:
I know I already approved this change in `gm-common`, but what's the difference between `characters` and `characterProperties`? Should they just be part of the same structure?

Reply:
`characters` is part of `constants`, so I'm not quite sure that would line up sensibly.

Reply:
Maybe we should name that `characterConstants` then? Idk, naming is too hard.

Reply:
🤷‍♂ works for me

Reply:
I'll leave it for now, since I don't want to go breaking everything that currently works.